wweval/R/ms_fig5.R

```diff
@@ -261,14 +261,14 @@ make_fig5_heatmap_relative_wis <- function(scores,
       model != {{ baseline_model }},
       location != "US",
       model %in% !!models_to_show
-    ) # exclued the US bc not available for
+    ) # exclude the US bc not available for
   # retro model
 
 
   p <- ggplot(relative_scores) +
     geom_tile(aes(x = model, y = short_name, fill = relative_interval_score)) +
     scale_fill_gradient2(
-      high = "red", mid = "white", low = "blue", transform = "log2",
+      high = "red", mid = "white", low = "blue",
```

Comment:
why was the transform removed?

Reply:
This must be an issue with my local vs VAP installed version of `ggplot2` -- I can fix this to make the versions match/update on the VAP. 

In my VAP installation of ggplot2, `transform` was not an argulemt to `scale_fill_gradient2()` so this was throwing an error. Should've looked into it more but just excluded to generate the figure

Reply:
You probably need to update ggplot2. It was changed from `trans` to `transform` in later version https://ggplot2.tidyverse.org/reference/scale_gradient.html